idaes/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -163,14 +167,14 @@ def build(self):
 
         # Volume of reactor
         self.bed_diameter = Var(initialize=1,
-                                doc='Reactor diameter [m]',
-                                units=pyunits.m)
+                                doc='Reactor diameter',
+                                units=units_meta_solid('length'))
         self.bed_height = Var(initialize=1,
-                              doc='Bed length [m]',
-                              units=pyunits.m)
+                              doc='Bed length',
+                              units=units_meta_solid('length'))
         self.volume_bed = Var(initialize=1.0,
-                              doc="Volume of the reactor bed [m**3]",
-                              units=pyunits.m**3)
+                              doc="Volume of the reactor bed",
+                              units=units_meta_solid('length')**3)
```

Comment:
For reference (don't fix now), you can do `derived_units("volume")` - a lot of common unit combinations are declared in the derived units metadata.